src/components/UI/MessagePanel/MessageToolsMenuContainer.vue

```diff
@@ -61,20 +46,26 @@ const useStyles = (
 
 export default defineComponent({
   name: 'MessageToolsMenuContainer',
-  components: {
-    ClickOutside,
-    MessageToolsMenu
+  components: { ClickOutside },
+  props: {
+    position: {
+      type: Object as PropType<{ x: number; y: number }>,
+      required: true
+    }
   },
-  setup() {
-    const { state, isShown, closeContextMenu } = useMessageContextMenuStore()
-    const position = toRef(state, 'position')
+  emits: {
+    closeContextMenu: () => true
+  },
+  setup(props, { emit }) {
+    const position = ref(props.position)
```

Comment:
今の状態だと`position`が変わったときにリアクティブが壊れますね 🖖 
```suggestion
    const position = toRef(props, 'position')
```